Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionsBubbleView+Default.swift

```diff
@@ -5,7 +5,7 @@
 import StreamChat
 import UIKit
 
-open class ChatMessageDefaultReactionsBubbleView: ChatMessageReactionsBubbleView {
+open class ChatMessageDefaultReactionsBubbleView: ReactionPickerBubbleView {
```

Comment:
I know this was like this before, but it doesn't make a lot of sense the current file name, it should just be:
`ChatMessageDefaultReactionsBubbleView.swift`